contrib/cirrus/pr-should-link-jira.t

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+#
+# tests for pr-should-link-jira.t
+#
+
+ME=$(basename $0)
+
+# Our test script queries github, for which we need token
+if [[ -z "$CIRRUS_REPO_CLONE_TOKEN" ]]; then
+    if [[ -n "$GITHUB_TOKEN" ]]; then
+       CIRRUS_REPO_CLONE_TOKEN="$GITHUB_TOKEN"
+    else
+        echo "$ME: Please set \$CIRRUS_REPO_CLONE_TOKEN" >&2
+        exit 1
+    fi
+fi
+
+export CIRRUS_REPO_CLONE_TOKEN
```

Comment:
I added it there so i could test it with:
```
CIRRUS_REPO_CLONE_TOKEN=... ./pr-should-link-jira.t
```
Otherwise it would not get propagated to the test.
That said, one might very well not want to support that...

Reply:
If you do want to keep this here, I would be helpful to add a comment suggesting it's needed for pr-should-link-jira.t to function.

Reply:
I moved it.